jest.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
   rootDir: ROOT_DIR,
   restoreMocks: true,
   reporters: ['default'],
-  modulePathIgnorePatterns: ['dist'],
+  modulePathIgnorePatterns: ['dist', '.bob'],
```

Comment:
related to this PR or leftovers?

Reply:
It is needed because`bob` compiles test files too and Jest runs them too